internal/cmd/server/start_alarm_subscription_server.go

```diff
@@ -219,14 +229,42 @@ func (c *AlarmSubscriptionServerCommand) run(cmd *cobra.Command, argv []string)
 		return exit.Error(1)
 	}
 
-	// Create the handler:
+	// Get the namespace:
+	o2imsNamespace, err := flags.GetString(namespaceFlagName)
+	if err != nil {
+		logger.DebugContext(
+			ctx,
+			"Failed to get o2ims namespace flag",
+			"flag", o2imsNamespace,
+			"error", err.Error(),
```

Comment:
Try to use the type safe interface of `slog`:

```go
loggger.DebugContext(
        ctx,
        "Failed to ....",
        slog.String("flag", namespaceFlagName),
        slog.Strng("error", err.Error(),
)
```

Note that the value of the "flag" here should be the flag name: the `o2imsNamespace` variable will always be empty because inside this block it failed to get it.